<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QMainWindow" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>561</height>
     </rect>
    </property>
    <widget class="QLabel" name="user">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 97, 203);</string>
     </property>
     <property name="text">
      <string>Usuário</string>
     </property>
    </widget>
    <widget class="QLabel" name="passWord">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 97, 203);</string>
     </property>
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usuario">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(192, 97, 203);
color: rgb(145, 65, 172);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="senha">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(145, 65, 172);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_entrar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(145, 65, 172);</string>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="no_conta">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 97, 203);
color: rgb(145, 65, 172);</string>
     </property>
     <property name="text">
      <string>Não tenho conta:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_criar_conta">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(129, 61, 156);</string>
     </property>
     <property name="text">
      <string>Criar conta</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 97, 203);</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
